### PR TITLE
V2 alk styles

### PR DESCRIPTION
**Pull request by adlenafane**

### CODE REVIEW

File: src/_colors.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@ $brand-inprogress:      #C69C5D;
 $brand-highlight:       #FABF2D;
 $brand-highlight-secondary: #EF7E1D;
 
-$brand-primary-hover:   #b2dae0;
+$brand-primary-hover:   #3e8e9a;
```
Comment (birdy-) [inline src/_colors.scss]: Are you sure ? (will change hover on list)

File: src/_colors.scss [hunk 2]
```diff
@@ -44,9 +44,10 @@ $brand-brillant:        rgb(255, 255, 0);
 $brand-deprecated:      rgb(165, 154, 103);
 $brand-stream:          $brand-primary;
 
-$alk-certified: #45b26e;
-$alk-accepted: #c79d59;
-$alk-attributed: #e74c3c;
+$alk-certified: $brand-success;
+$alk-accepted: $brand-inprogress;
+$alk-attributed: $brand-danger;
+$alk-discontinued: $gray-light;
```
Comment (birdy-) [inline src/_colors.scss]: Are you sure ?

File: src/_colors.scss (hunk 1 above)
Comment (adlenafane) [inline src/_colors.scss]: Yeah I checked it this morning with @switchalkemics and tita

**Comment by birdy-** [comment]
LGTM

File: src/_colors.scss (hunk 1 above)
Comment (adlenafane) [inline src/_colors.scss]: Ok fixed

File: src/_colors.scss (hunk 2 above)
Comment (adlenafane) [inline src/_colors.scss]: Yep just agreed

**Pull request merged**